src/components/datetime/editForm/DateTime.edit.date.js

```diff
@@ -10,23 +10,79 @@ export default [
     weight: 0,
     tooltip: 'Enables date input for this field.'
   },
+  {
+    type: 'checkbox',
```

Comment:
Can we make this non-persistent so it won't get saved with the component JSON?

Reply:
@travist Do you mean I should move customConditional to component from json and add it automatically when component creates?

Reply:
No, I just mean to add the persistent flag to this component.  ```persistent: false``` or maybe it should be ```persistent: 'client-only'```.